FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			completion(Swift.Result { try Self.mapClientResult(result) })
+		}
+	}
+
+	private static func mapClientResult(_ clientResult: HTTPClient.Result) throws -> [FeedImage] {
+		switch clientResult {
+		case .failure:
+			throw Error.connectivity
+		case .success((let data, let httpResponse)):
```

Comment:
The `let` can be defined outside the tuple so the expression can be shorter:
```suggestion
		case let .success((data, httpResponse)):
```